src/sku_scrape/scraper.clj

```diff
@@ -0,0 +1,70 @@
+(ns sku-scrape.scraper
+  (:require [clj-http.client :as http]
+            [crouton.html :as html]
+            [net.cgrand.enlive-html :as enlive]))
+
+(def platforms {:amazon
+                {:query-link            "https://www.amazon.in/s?k="
+                 :results-selector      ;[:div.a-section :div.sg-row :div.sg-col-4-of-12.sg-col-8-of-16.sg-col-16-of-24.sg-col-12-of-20.sg-col-24-of-32.sg-col-28-of-36.sg-col-20-of-28 :> :div.sg-col-inner]
+                 [:.sg-col-4-of-12.sg-col-8-of-16.sg-col-16-of-24.sg-col-12-of-20.sg-col-24-of-32.sg-col.sg-col-28-of-36.sg-col-20-of-28]
+                 :result-deconstruction {:price     {:selector    [:.a-price-whole]
+                                                     :constructor [:content 0]}
+                                         :currency  {:selector    [:.a-price-symbol]
+                                                     :constructor [:content 0]}
+                                         :link      {:selector    [:h2 (enlive/attr? :href)]
+                                                     :constructor [:attrs :href]}
+                                         :link-text {:selector    [:h2 (enlive/attr? :href)]
+                                                     :constructor [:content 1 :content 0]}}}
+                :flipkart
+                {:query-link            "https://www.flipkart.com/search?q="
+                 :results-selector      [:.col-12-12 :> :div :> :div]
+                 :result-deconstruction []}
+                :shopclues
+                {:query-link            "https://www.shopclues.com/search?q="
+                 :results-selector      []
+                 :result-deconstruction []}
+                :myntra
+                {:query-link            "https://www.myntra.com/"
+                 :results-selector      []
+                 :result-deconstruction []}})
+
```

Comment:
the selectors for enlive are very finicky. works sometimes, doesnt work 90% of the time